css/about.css

```diff
@@ -0,0 +1,199 @@
+.container{
+    background: #1C2027;
+    padding: 2rem 6rem;
```

Comment:
Please make the x-axis padding 10rem or use the exact padding measurement you used for the home page